.go-version

```diff
@@ -1 +1 @@
-go1.17.11
+1.17.11
```

Comment:
nice catch @joshua-goldstein ... now the tests are running (and progressing further than before)